bookwyrm/settings.py

```diff
@@ -370,3 +370,8 @@
 HTTP_X_FORWARDED_PROTO = env.bool("SECURE_PROXY_SSL_HEADER", False)
 if HTTP_X_FORWARDED_PROTO:
     SECURE_PROXY_SSL_HEADER = ("HTTP_X_FORWARDED_PROTO", "https")
+
+# AUTHORIZED_FETCH Instance Actor
+# WARNING this must both be unique - not used by any other user
+INSTANCE_ACTOR_USERNAME = DOMAIN
+INSTANCE_ACTOR_EMAIL = f"representative@{DOMAIN}"
```

Comment:
I would suggest a total random string instead of `representative`.

My thoughts behind that suggestion:
An admin updates the production version with `git pull` and there is already a user with the name `representative` the name is not unique any more. I know, one should read the update notes before updating but in this case the admin would need toward around this. This breaks the communicated update process and is a pitfall for those who are not reading patch notes. 

If BookWyrm introduces a new way to update an instance there should be a communication within the instance for admins and a documentation on how to perform the update.

Reply:
I think this used to be `DOMAIN@DOMAIN` in mastodon and now is `mastodon.internal`: https://github.com/mastodon/mastodon/blob/main/app/models/concerns/account_finder_concern.rb#L20
I think something like `bookwyrm.internal` should be reserved in any case, but indeed representative might be a common-enough name

Reply:
As currently written the username is DOMAIN@DOMAIN - it's the email address (which presumably will never be used) that is representative@DOMAIN.

I'm not really fussed what we use, other than agreeing it needs to be unlikely to be currently in use. the suggestion of `bookwyrm.internal@DOMAIN` seems good.

It also occurred to me when working on this that we should give admins the ability to reserve a bunch of usernames they don't want people to be able to register  (e.g. "admin" or "support" or whatever they want). I'll make a separate Issue for that.